services/device-connection/src/main/java/org/eclipse/hono/deviceconnection/infinispan/RemoteCacheBasedDeviceConnectionService.java

```diff
@@ -45,7 +44,6 @@
  */
 public class RemoteCacheBasedDeviceConnectionService extends EventBusDeviceConnectionAdapter implements DeviceConnectionService, HealthCheckProvider {
 
-    private static final Logger LOG = LoggerFactory.getLogger(RemoteCacheBasedDeviceConnectionService.class);
```

Comment:
I think it is helpful to know that problems originate in this class.

Reply:
IMHO the *log* instance does provide for this as it is created using the current class name:
```
    /**
     * A logger to be shared by subclasses.
     */
    protected final Logger log = LoggerFactory.getLogger(getClass());
```

Reply:
Ok, what I meant was: that I think it is helpful to know that a log/problem originates in this, or parent/sub-classes.

Right now, also messages from e.g. `EventBusDeviceConnectionAdapter` will have the name of `RemoteCacheBasedDeviceConnectionService`. So if you see a message in the log, you always need to walk through the class hierarchy, in order to find the right source.

So should there be sub-classes of this class, the would all log messages from this class, under a different name.